pyproject.toml

```diff
@@ -46,6 +39,46 @@ tests = [
     "pytest >=6.0",
     "pytest-dotenv",
     "requests-mock",
+    "apache-airflow-providers-google",
+    "sqlalchemy-bigquery==1.3.0",
+    "smart-open[all]>=5.2.1",
+    "apache-airflow-providers-snowflake",
+    "snowflake-sqlalchemy ==1.2.0",
+    "snowflake-connector-python[pandas]",
+    "apache-airflow-providers-postgres",
+    "s3fs",
+    "smart-open[s3]>=5.2.1",
+]
+gcs = [
```

Comment:
Do these dependencies need to be within a `[project.optional-dependencies]` block?
https://www.python.org/dev/peps/pep-0631/#optional-dependencies

Reply:
@uranusjr can you please confirm this?

Reply:
Having a group in `optional-dependencies` (as is structured now) is enough to expose those as extras. The `pip install` should work.